test/opt/fold_test.cpp

```diff
@@ -3297,7 +3297,16 @@ INSTANTIATE_TEST_SUITE_P(IntegerArithmeticTestCases, GeneralInstructionFoldingTe
             "%2 = OpIMul %int %3 %int_1\n" +
             "OpReturn\n" +
             "OpFunctionEnd",
-        2, 3)
+        2, 3),
+    // Test case 20: Don't fold comparisons of 64-bit types
```

Comment:
This looks like test case 42.  I do that to make it easy to find the test case when it fails.  The output will tell you the test number of the test that failed.

Reply:
Ah, yes, I had initially tried to make it a bool folding test, but that did not work well.